torchtitan/distributed/pipeline.py

```diff
@@ -39,74 +38,74 @@ def generate_split_points(
 
     Args:
         schedule_str (str): The string of the schedule name.
-        layers_per_stage (int): The number of layers per stage.
-        pp_dim (int): The pipeline parallel dimension.
+        pp_degree (int): The pipeline parallel dimension.
         num_layers (int): The number of layers in the model.
-        input_output_weight (int): The number of layers to consider the input/output modules in the layer calculation.
+        input_weight (int): The number of layers to consider the input modules in layer calculation.
+        output_weight (int): The number of layers to consider the output modules in layer calculation.
+        num_layers_per_virtual_stage (int): The number of layers per virtual pipeline stage.
```

Comment:
nit: the name `num_layers_per_stage` is clearer to me, not sure if people will understand what "virtual" means in this case.